pkgs/applications/networking/browsers/firefox/packages.nix

```diff
@@ -40,64 +40,6 @@ rec {
     };
   };
 
-  # Do not remove. This is the last version of Firefox that supports
-  # the old plugins. While this package is unsafe to use for browsing
-  # the web, there are many old useful plugins targeting offline
-  # activities (e.g. ebook readers, syncronous translation, etc) that
-  # will probably never be ported to WebExtensions API.
-  firefox-esr-52 = (common rec {
-    pname = "firefox-esr";
-    ffversion = "52.9.0esr";
-    src = fetchurl {
-      url = "mirror://mozilla/firefox/releases/${ffversion}/source/firefox-${ffversion}.source.tar.xz";
-      sha512 = "bfca42668ca78a12a9fb56368f4aae5334b1f7a71966fbba4c32b9c5e6597aac79a6e340ac3966779d2d5563eb47c054ab33cc40bfb7306172138ccbd3adb2b9";
-    };
-
-    patches = [
-      # this one is actually an omnipresent bug
-      # https://bugzilla.mozilla.org/show_bug.cgi?id=1444519
-      ./fix-pa-context-connect-retval.patch
-    ];
-
-    meta = firefox.meta // {
-      description = "A web browser built from Firefox Extended Support Release source tree";
-      knownVulnerabilities = [ "Support ended in August 2018." ];
-    };
-  }).override {
-    stdenv = overrideCC gccStdenv gcc6; # gcc7 fails with "undefined reference to `__divmoddi4'"
-    gtk3Support = false;
-  };
-
-  firefox-esr-60 = common rec {
-    pname = "firefox-esr";
-    ffversion = "60.9.0esr";
-
-    src = fetchurl {
-      url = "mirror://mozilla/firefox/releases/${ffversion}/source/firefox-${ffversion}.source.tar.xz";
-      sha512 = "4baea5c9c4eff257834bbaee6d7786f69f7e6bacd24ca13c2705226f4a0d88315ab38c650b2c5e9c76b698f2debc7cea1e5a99cb4dc24e03c48a24df5143a3cf";
-    };
-
-    patches = [
-      ./no-buildconfig-ffx65.patch
-
-      # this one is actually an omnipresent bug
-      # https://bugzilla.mozilla.org/show_bug.cgi?id=1444519
-      ./fix-pa-context-connect-retval.patch
-
-      missing-documentation-patch
-    ];
-
-    meta = firefox.meta // {
-      description = "A web browser built from Firefox Extended Support Release source tree";
-      knownVulnerabilities = [ "Support ended around October 2019." ];
-    };
-    updateScript = callPackage ./update.nix {
-      attrPath = "firefox-esr-60-unwrapped";
-      versionSuffix = "esr";
-      versionKey = "ffversion";
-    };
-  };
-
   firefox-esr-68 = common rec {
```

Comment:
@andir WDYT about just calling this `firefox-esr`, and have it point to the latest currently supported ESR release?

Reply:
There are times where there are two overlapping ESRs (for a few months). I'd prefer if we give people the option to slowly transition between them once a new ESR has been released. I would just keep it the way we've been doing it (for now).

Reply:
Okay.